lib/internal/fs/utils.js

```diff
@@ -173,6 +173,29 @@ function copyObject(source) {
   return target;
 }
 
+function join(path, name) {
+  if ((typeof path === 'string' || path instanceof Buffer) &&
```

Comment:
Can you use `isUint8Array(path)` instead of `path instanceof Buffer` here, like we do elsewhere in this file? We generally accept any kind of `Uint8Array` in all the places where our API accepts `Buffer`s.

Reply:
done